udata_gouvfr/theme/gouvfr/templates/header.html

```diff
@@ -32,7 +32,7 @@
                         @click.capture.prevent="$refs.suggest.startSearch()">
                         {% include theme('svg/search.svg') %}
                     </a>
-                    <a href="#" @click.capture.prevent="$refs.header.classList.add('mobile-menu-open')"
+                    <a href="#" @click.capture.prevent="$refs.header.classList.add('mobile-menu-open'); if ($refs.suggest.active) {$refs.suggest.stop();}"
```

Comment:
```suggestion
                    <a href="#" @click.capture.prevent="$refs.header.classList.add('mobile-menu-open'); if ($refs.suggest.active) $refs.suggest.stop();"
```

Reply:
Maybe add a comment to explain what this does, because I have no idea right now :-)

Reply:
Yep, I can add a comment, but it is supposed to deactivate the suggest view when opening the burger menu.